### PR TITLE
completed prep-example exercise

### PR DESCRIPTION
**Pull request by HMerizan**
Screen Shot 2021-09-18 at 1 41 30 PM

### CODE REVIEW

File: prep-example/index.html [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
+
+
```
Comment (thebearingedge) [inline prep-example/index.html]: Hi Heather! Looks like your custom content was not included in the file. Be sure to let me know if I can help out!

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: prep-example/index.html (hunk 1 above)
Comment (HMerizan) [reply prep-example/index.html]: I am now stuck in the corrections.  It told me it was an invalid username and password and that authentication failed.
Screen Shot 2021-09-20 at 6 34 35 PM

Comment (HMerizan) [reply prep-example/index.html]: Screen Shot 2021-09-20 at 6 11 54 PM

**Review by thebearingedge — APPROVED** [review]
Awesome 👍

**Pull request merged**